server/src/controllers/noncommercial.es6

```diff
@@ -288,73 +345,74 @@ noncommercial.create = (req, res) => {
     errorRet['errors'] = errorArr;
     res.status(400).json(errorRet);
   } else {
-    // create the noncommercial app object and persist
     util.setAuthEmail(req);
-    NoncommercialApplication.create(translateFromClientToDatabase(req.body))
+    let model = {
+      authEmail: req.body.authEmail
+    };
+    translateFromClientToDatabase(req.body, model);
+    NoncommercialApplication.create(model)
       .then(noncommApp => {
         email.sendEmail('noncommercialApplicationSubmittedAdminConfirmation', noncommApp);
         email.sendEmail('noncommercialApplicationSubmittedConfirmation', noncommApp);
         req.body['applicationId'] = noncommApp.applicationId;
         req.body['appControlNumber'] = noncommApp.appControlNumber;
-        res.status(201).json(req.body);
+        return res.status(201).json(req.body);
       })
       .catch(error => {
-        res.status(500).json(error);
+        return res.status(500).json(error);
       });
   }
 };
 
 noncommercial.update = (req, res) => {
```

Comment:
it feels like this call has a high cyclomatic complexity... any way to bring this out?